### PR TITLE
"Redeploying Master Certificates Only" recreates "Service Serving Certificates" CA Certificate

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1550250

### CODE REVIEW

File: install_config/redeploying_certificates.adoc [hunk 1]
```diff
@@ -385,9 +385,17 @@ file:
 
 ----
 $ ansible-playbook -i <inventory_file> \
-    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-certificates.yml
+    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-master-certificates.yml
```
Comment (gpei) [inline install_config/redeploying_certificates.adoc]: The playbook name should not be changed. The previous one is correct for ocp-3.9.
What we may need to improve is "*_redeploy-master-certificates.yml_* playbook" in line380, for more specifically, we should name it as "*_master redeploy-certificates.yml_* playbook".
So is in line 402 and line 421, which should be:
The *_etcd redeploy-certificates.yml_* playbook
The *_node redeploy-certificates.yml_* playbook

File: install_config/redeploying_certificates.adoc [hunk 2]
```diff
@@ -385,9 +385,17 @@ file:
...
 ----
 
+[IMPORTANT]
```
Comment (gpei) [inline install_config/redeploying_certificates.adoc]: This part lgtm :+1:

**Comment by mburke5678** [comment]
@gpei I changed the `redeploy-master-certificates.yml` back to `redeploy-certificates.yml`. Apologies. Not sure why I made that change.

**Comment by gpei** [comment]
@mburke5678 For 3.7 and previous version, the playbook used for redeploying master cert is `playbooks/byo/openshift-cluster/redeploy-master-certificates.yml`
From 3.9, it changed as
`playbooks/openshift-master/redeploy-certificates.yml`
So there's no `redeploy-master-certificates.yml` file in 3.9

**Comment by mburke5678** [comment]
@gpei Does this look good to merge to 3.9 and 3.10? 
I can manually add the note to 3.7 and previous versions back to 3.3. Is this an acceptable plan?

**Comment by gpei** [comment]
@mburke5678 sorry, I'm not able to catch any new changes in this PR.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL Small change

File: install_config/redeploying_certificates.adoc [hunk 3]
```diff
@@ -385,9 +385,17 @@ file:
...
+====
+After executing this playbook, you need to regenerate any xref:../dev_guide/secrets.adoc#service-serving-certificate-secrets[service signing certificate/key pairs]
```
Comment (kalexand-rh) [inline install_config/redeploying_certificates.adoc]: executing should be running
certificate/key pairs should be certificates or key pairs

File: install_config/redeploying_certificates.adoc [hunk 4]
```diff
@@ -385,9 +385,17 @@ file:
...
 ----
 $ ansible-playbook -i <inventory_file> \
-    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-certificates.yml
+    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-master-certificates.yml
 ----
 
+[IMPORTANT]
+====
+After executing this playbook, you need to regenerate any xref:../dev_guide/secrets.adoc#service-serving-certificate-secrets[service signing certificate/key pairs] 
+by deleting existing secrets containing service serving certificates or removing and re-adding
```
Comment (kalexand-rh) [inline install_config/redeploying_certificates.adoc]: containing should be that contain
Is there another way to say "service serving certificates?"

Comment (mburke5678) [reply install_config/redeploying_certificates.adoc]: @kalexand-rh It appears that  "service serving certificates" was the term when the feature was added to OpenShift. 
https://github.com/openshift/openshift-docs/pull/2324/files

Comment (kalexand-rh) [reply install_config/redeploying_certificates.adoc]: @mburke5678, that's fair. A bit confusing still, but fair. :)

**Pull request merged**